test/libsolidity/syntaxTests/types/hex_string_to_bytesNN_same_size_explicit.sol

```diff
@@ -0,0 +1,13 @@
+contract C {
```

Comment:
These files are copies of the existing tests which use decimals and hexadecimals.